accounts/views.py

```diff
@@ -1069,3 +1065,33 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
+    if settings.DEBUG:
+        paypal_validation_url = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+    else:
+        paypal_validation_url = "https://www.paypal.com/cgi-bin/webscr"
+    r = requests.post(paypal_validation_url, data=params)
+    if r.text == 'VERIFIED':
+        Donation.objects.get_or_create(transaction_id=params['txn_id'], defaults={
+        'email': params['payer_email'],
+        'display_name': params['custom'],
+        'amount': params['mc_gross'],
+        'currency': params['mc_currency']})
+    return HttpResponse("OK")
+
+
+def donate(request):
+    tvars = {'paypal_email': settings.PAYPAL_EMAIL}
```

Comment:
``tvars`` is the name we decided to use for the dictionary of variables passed to templates. When we did the refactoring of accounts and sounds module (and probably some others) and stopped using ``render_to_template`` in favour of ``render``, we started using ``tvars`` as the dictionary that is almost always built before calling ``render`` and passed as a parameter. Before we were most of the times using ``locals()`` or simply building the dictionary in line with the ``render_to_template``call.